src/content/docs/implementation/marketing/commerce-instrumentation.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Commerce instrumentation
+description: Learn how to instrument your Adobe Commerce site for marketing and analytics.
+---
+
+For Live Search and Product Recommendation APIs to work correctly, you must collect and send user interaction events to Commerce.
+
+User interaction evets power Adobe Sensei features, like intelligent merchandising for Live Search and to personalize product recommendations units. These events also populate performance dashboards for [Live Search](https://experienceleague.adobe.com/en/docs/commerce-merchant-services/live-search/live-search-admin/performance) and [Product Recommendations](https://experienceleague.adobe.com/en/docs/commerce-merchant-services/product-recommendations/admin/workspace).
+
+## Adobe Client Data Layer (ACDL)
+
+The boilerplate includes the [Adobe Client Data Layer (ACDL)](https://github.com/adobe/adobe-client-data-layer) by default. It uses a custom, more lightweight version that is suitable for loading early in Edge Delivery Services.
```

Comment:
It's a bit weird linking to the main ACDL repo and then saying that boilerplate is using something better without specifying where this better version is :) 

The version boilerplate is using is here https://github.com/adobe/adobe-client-data-layer/tree/validator.

Since this wasn't merged and nobody takes ownership of that project. Let's change this and just say that boilerplate uses ACDL for instrumentation.